skynet/main.py

```diff
@@ -46,6 +46,15 @@ async def lifespan(main_app: FastAPI):
         main_app.mount('/assistant', rag_app)
         await assistant_startup()
 
+    if 'customerconfigs' in modules:
```

Comment:
not a fan of the combined wording, i would prefer the mounting path to be `customer-config`